arrow-site/doc/docs/arrow-core/datatypes/either/README.md

```diff
@@ -34,9 +34,9 @@ val magic = throwsSomeStuff.andThen(throwsOtherThings).andThen(moreThrowing)
 magic
 ```
 
-Assume we happily throw exceptions in our code. Looking at the types of the above functions, any of them could throw any number of exceptions -- we do not know. When we compose, exceptions from any of the constituent
+Assume we happily throw exceptions in our code. Looking at the types of functions above, any could throw a number of exceptions -- we do not know. When we compose, exceptions from any of the constituent
 functions can be thrown. Moreover, they may throw the same kind of exception
-(e.g. `IllegalArgumentException`) and thus it gets tricky tracking exactly where an exception came from.
+(e.g., `IllegalArgumentException`) and, thus, it gets tricky tracking exactly where an exception came from.
```

Comment:
This is awesome stuff... I was noticing the lack of commas in the docs while browsing them, so I'm glad someone took the time to rectify this. I'd question the grammar here, though. I think it should read:

Moreover, they may throw the same kind of exception (e.g., `IllegalArgumentException`), and thus, it gets tricky tracking exactly where an exception came from.